import-service/handlers/import-file-parser/index.ts

```diff
@@ -0,0 +1,33 @@
+import {S3Handler} from 'aws-lambda';
+import 'source-map-support/register';
+import * as AWS from 'aws-sdk';
+import {logLambdaArgs} from '../../../shared/utils';
+import {parseS3CsvObj} from "./parse-s3-csv-obj";
+import {copyAndDeleteS3Obj} from "./copy-and-delete-s3-obj";
+
+export const importFileParser: S3Handler = async (event, _context) => {
```

Comment:
From the task description:
![image](https://user-images.githubusercontent.com/41877460/99257472-220cd000-2828-11eb-91b2-60069575f5f6.png)

Please add a response here

Reply:
@AndreyS55 S3Handler should not return HTTP response:
`export type S3Handler = Handler<S3Event, void>;`
`export type Handler<TEvent = any, TResult = any> = (
    event: TEvent,
    context: Context,
    callback: Callback<TResult>,
) => void | Promise<TResult>;`